platform/vs/docker-sonic-vs/platform.json

```diff
@@ -3,194 +3,350 @@
         "Ethernet0": {
             "index": "0,0,0,0",
             "lanes": "25,26,27,28",
-            "alias_at_lanes": "fortyGigE0/0,fortyGigE0/1,fortyGigE0/2,fortyGigE0/3",
-            "breakout_modes": "1x100G[40G],2x50G,4x25G[10G],2x25G(2)+1x50G(2),1x50G(2)+2x25G(2)"
+            "breakout_modes": {
+                "1x100G[40G]": ["fortyGigE0/0"],
+                "2x50G": ["fortyGigE0/0", "fortyGigE0/1"],
+                "4x25G[10G]": ["fortyGigE0/0", "fortyGigE0/1", "fortyGigE0/2", "fortyGigE0/3"],
+                "2x25G(2)+1x50G(2)": ["fortyGigE0/0", "fortyGigE0/1", "fortyGigE0/2"],
+                "1x50G(2)+2x25G(2)": ["fortyGigE0/0", "fortyGigE0/1", "fortyGigE0/2"]
+            }
         },
        "Ethernet4": {
             "index": "1,1,1,1",
             "lanes": "29,30,31,32",
-            "alias_at_lanes": "fortyGigE0/4,fortyGigE0/5,fortyGigE0/6,fortyGigE0/7",
-            "breakout_modes": "1x100G[40G],2x50G,4x25G[10G]"
+            "breakout_modes": {
+                "1x100G[40G]": ["fortyGigE0/4"],
+                "2x50G": ["fortyGigE0/4", "fortyGigE0/5"],
```

Comment:
should be 4 and 6.

Reply:
Shouldn't it be fortyGigE1/1 and  fortyGigE1/3 instead?

Reply:
Again, this is for vs tests, I am not sure if we should stuck with one way or the other.  The design is flexible.

Reply:
it is correctness, i think ying is correct. @zhenggen-xu, can you address as ying's suggestion?

Reply:
OK, this PR is just following the previous convention. Let's fix that all together here. For vs tests, since we don't really care the alias value,  if you guys strongly prefer the jumping name for 2x50G, we will make that change.  For the real HW platform, we will keep alias to be consecutive even for 2x50G.

Reply:
@yxieca , my suggestion is correct, yours is incorrect.

Reply:
OK, it was following https://github.com/Azure/sonic-buildimage/blob/201811/device/dell/x86_64-dell_s6000_s1220-r0/Force10-S6000/port_config.ini . Here we will just to make 2x50G separated.

Reply:
Modified as requested. 
`"2x50G": ["fortyGigE0/4", "fortyGigE0/6"]`